src/Raven.Server/Web/System/ShardedAdminDatabaseHandler.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Raven.Client.Exceptions.Database;
+using Raven.Client.ServerWide;
+using Raven.Client.ServerWide.Sharding;
+using Raven.Client.Util;
+using Raven.Server.Routing;
+using Raven.Server.ServerWide.Commands;
+using Raven.Server.ServerWide.Context;
+using Raven.Server.Utils;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Web.System
+{
+    public class ShardedAdminDatabaseHandler : ServerRequestHandler
+    {
+        [RavenAction("/admin/orchestrator", "PUT", AuthorizationStatus.Operator)]
```

Comment:
let's keep convention and change that to `/admin/databases/orchestrator` ? Does it make sense? Other endpoint is `/admin/databases/node` IIRC

Reply:
Yep. Changed